assets/js/app.js

```diff
@@ -5,8 +5,22 @@ import {Socket} from "phoenix"
 import {LiveSocket} from "phoenix_live_view"
 import topbar from "../vendor/topbar"
 
+
+let Hooks = {}
 let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
-let liveSocket = new LiveSocket("/live", Socket, {params: {_csrf_token: csrfToken}})
+
+let liveSocket = new LiveSocket("/live", Socket, {
+  hooks: Hooks,
+  dom:{
+        onBeforeElUpdated(from, to) {
```

Comment:
What does this do? 🤷‍♂️

Reply:
![image](https://user-images.githubusercontent.com/6057298/204491385-345ffc60-4fe7-4cdb-b906-be8cd0107cef.png)

see https://hexdocs.pm/phoenix_live_view/js-interop.html#client-hooks-via-phx-hook

This is required to make sure Alpine.js and LiveView can work together

Reply:
see also: https://github.com/dwyl/learn-alpine.js/blob/main/drag-and-drop.md#4-drag-and-drop

Reply:
Your reply on this PR comment will be lost to the ether and anyone reading this code in the future will be clueless.